### PR TITLE
Fixer on structured benchmark

### LINKED ISSUE #70: Adapt code fixing to the new benchmark YAML
Related: https://github.com/google/oss-fuzz-gen/issues/54

### PR DESCRIPTION
**Pull request by cjx10**
Resolves #70

### CODE REVIEW

File: experiment/builder_runner.py [hunk 1]
```diff
@@ -88,27 +88,23 @@ def _contains_target_function(self, target_path: str) -> bool:
...
-           f'`{self.benchmark.function_signature}` INSIDE FUNCTION '
-           '`LLVMFuzzerTestOneInput`.')
-      ]
+      build_result.errors = [(
+          f'The target function <code>{self.benchmark.function_signature}</code>'
+          ' was not called by the fuzz target '
+          '<code>LLVMFuzzerTestOneInput</code>.'
+          'YOU MUST CALL FUNCTION '
+          f'<code>{self.benchmark.function_signature}</code> INSIDE FUNCTION '
+          '<code>LLVMFuzzerTestOneInput</code>.')]
```
Comment (DonggeLiu) [inline experiment/builder_runner.py:107]: nit: I reckon this is also a prompt fix that belongs to the other PR?

File: experiment/builder_runner.py [hunk 2]
```diff
@@ -88,27 +88,23 @@ def _contains_target_function(self, target_path: str) -> bool:
     """Validates if the LLM-generated code contains the target function."""
     with open(target_path) as generated_code_file:
       generated_code = generated_code_file.read()
-    target_function_name = self.benchmark.function_name
-    return bool(
-        re.search(
-            benchmarklib.function_name_regex(target_function_name,
-                                             include_top_level=True),
-            generated_code))
+    # Get the top level function name without namespace
```
Comment (DonggeLiu) [inline experiment/builder_runner.py:91]: nit: Ends a sentence with a period (.).

**Review by DonggeLiu — APPROVED** [review]

File: experiment/builder_runner.py (hunk 1 above)
Comment (cjx10) [reply experiment/builder_runner.py]: Yes indeed, moved.

Comment (DonggeLiu) [reply experiment/builder_runner.py]: Thanks! merged.

**Pull request merged**